README.md

```diff
@@ -1,6 +1,6 @@
-## Welcome to GitHub Games
+## Welcome to awesome game of Tetris
 
-A project based learning activity for people who are getting started with Git and GitHub.
+A project based learning activity for people who are getting started with Git and GitHub which is also a demo I did to complete this tutorial
 
 To play the game:
```

Comment:
Left reviews and line comments
Created a file and made a commit
Opened a pull request